### PR TITLE
fixed remaining largest leaks in kean tests

### PR DESCRIPTION
**Pull request by sebastianbaginski**
Related to strings in "CreateAlphaByteStringTest"
@marcusnaslund

### CODE REVIEW

File: test/draw/CreateAlphaByteStringTest.ooc [hunk 1]
```diff
@@ -20,9 +20,11 @@ CreateAlphaByteStringTest: class extends Fixture {
 			file := File new(filename)
 			folder := file parent . mkdirs() . free()
 			file free()
-			fw := FileWriter new(filename + "DataFile.ooc")
+			filename = filename >> "DataFile.ooc"
+			fw := FileWriter new(filename)
 			fw write(outputData)
 			fw close()
+			fw free()
```
Comment (marcusnaslund) [inline test/draw/CreateAlphaByteStringTest.ooc]: Let's do `fw write(outputData) . close() . free()` :)

File: source/draw/CreateAlphaByteString.ooc [hunk 2]
```diff
@@ -22,16 +22,17 @@ CreateAlphaByteString: class {
...
 			intString := ip[i] toString()
 			imageArray = imageArray >> "," >> (i % 32 == 0 ? "\n" : " ") >> intString
 			intString free()
 		}
-		imageArray += "]"
-		result := name + "Image" + ": StaticOverlayImages\n" + name + "Image image = " + imageArray + "\n"
-		result += name + "Image size = IntVector2D new(" + image size x toString() + ", " + image size y toString() + ")\n"
+		imageArray = imageArray >> "]"
+		result := (name + "Image: StaticOverlayImages\n") & (name + "Image image = ") & (imageArray >> "\n")
+		result = (result >> name) & ("Image size = IntVector2D new(" << image size x toString()) & (", " << image size y toString()) >> ")\n"
```
Comment (marcusnaslund) [inline source/draw/CreateAlphaByteString.ooc]: Wouldn't something like `result := "%s + Image: StaticOverlayImages\n%sImage image=%s\n%sImage size = IntVector2D new(%d, %d)\n format(name, name, imageArray, name, image size x, image size y)` look just a little bit better?

Comment (marcusnaslund) [inline source/draw/CreateAlphaByteString.ooc]: With reservation for any mistakes in that code.

Comment (sebastianbaginski) [inline source/draw/CreateAlphaByteString.ooc]: Yes, looks better

Comment (sebastianbaginski) [inline source/draw/CreateAlphaByteString.ooc]: ... but causes segmentation fault :(

**Comment by sebastianbaginski** [comment]
Updated according to the first comment. For the other one, I don't feel like debugging segmentation fault related to string formatting now.

**Comment by marcusnaslund** [comment]
No, I guess if it works well and leaks less than before, it's OK for now.

**Comment by sebastianbaginski** [comment]
@marcusnaslund is there something more you'd like me to do here before you move it to "final review" ?

**Comment by marcusnaslund** [comment]
Sorry.

**Pull request merged**